### PR TITLE
Update alt_ip, add dashnode2, add lisknode5 and liskservice5

### PR DESCRIPTION
**Pull request by confleux**

### CODE REVIEW

**Review by adamant-al — CHANGES_REQUESTED** [review]
- Update `infoService`
- Add `sunshine` node
- Add `alt_ip` for `node1.adamant.business`

File: assets/general/lisk/info.json [hunk 1]
```diff
@@ -28,15 +28,28 @@
...
+        "url": "https://liskservice3.adamant.im",
+        "alt_ip": "http://157.90.229.236:9904"
+      },
+      {
+        "url": "https://liskservice4.adamant.im",
+        "alt_ip": "http://78.47.205.206:9904"
+      }
     ]
   },
+
```
Comment (bludnic) [inline assets/general/lisk/info.json:52]: extra breakline, prettify it

Comment (bludnic) [reply assets/general/lisk/info.json:52]: npx prettier . --write

**Review by bludnic — CHANGES_REQUESTED** [review]

**Review by bludnic — APPROVED** [review]

**Review by adamant-al — APPROVED** [review]

**Comment by bludnic** [comment]
As we are migrating to Lisk v4, the old nodes (lisknode3, lisknode4) must be removed temporarily from the config. Same story for Lisk services.

After we release the PWA and iOS, we should update remaining nodes to the latest version and rollback the full list of nodes.

File: assets/general/dash/info.json [hunk 2]
```diff
@@ -29,7 +29,16 @@
     "oldPendingAttempts": 4
   },
   "txConsistencyMaxTime": 800000,
-  "nodes": [{ "url": "https://dashnode1.adamant.im" }],
+  "nodes": [
+    {
+      "url": "https://dashnode1.adamant.im",
+      "alt_ip": "http://95.216.208.142:44099"
```
Comment (adamant-al) [inline assets/general/dash/info.json:35]:
```suggestion
      "alt_ip": "http://45.85.147.224:44099"
```

**Review by adamant-al — APPROVED** [review]
@confleux 
Changed dashnode1 IP. Does the port stay the same?

**Pull request merged**